homeassistant/components/plex/server.py

```diff
@@ -674,6 +674,95 @@ def lookup_media(self, media_type, **kwargs):
             _LOGGER.error("%s not found in %s", failed_item, library_name)
             return None
 
+    @staticmethod
+    def continue_media(media, continuation):
```

Comment:
Instead of having a static method on the `PlexServer` object, maybe this method would fit better in `media_search.py`?

Reply:
done, and tests were moved accordingly to stay consistent